### PR TITLE
Learnable Model Create

### PR DESCRIPTION
**Pull request by Luckygyana**

### CODE REVIEW

File: trainingmodel.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    def forward(self, input):
+        return input * self.scale
+
+
+class AudioJNDModel(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.model = torchopenl3.models.load_audio_embedding_model(
+            input_repr="mel256", content_type="music", embedding_size=6144
```
Comment (turian) [inline trainingmodel.py]: I said "env" not "music" I thought

File: trainingmodel.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        )
+        self.scale = ScaleLayer()
+        self.cos = nn.CosineSimilarity(eps=1e-6)
+
+    def forward(self, x1, x2):
+        bs, in1, in2, in3 = x1.size()
+        x1 = self.model(x1.view(-1, in2, in3))
+        x2 = self.model(x2.view(-1, in2, in3))
+        x1 = self.scale(x1).view(bs, -1)
+        x2 = self.scale(x2).view(bs, -1)
```
Comment (turian) [inline trainingmodel.py]: Are you sure these views are correct?

Comment (Luckygyana) [reply trainingmodel.py]: Yes I double checked

File: trainingmodel.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        prob = self.cos(x1, x2)
+        assert torch.all((prob < 1) & (prob > -1))
+        prob = (prob + 1) / 2
+        assert torch.all((prob < 1) & (prob > 0))
+        return prob
+
+
+if __name__ == "__main__":
+    input1 = torch.randn(2, 17, 1, 48000)  # 2 is here batch_size
```
Comment (turian) [inline trainingmodel.py]: Okay but we're starting from audio files that have shape (n_samples) so how do we get them to the framed shape? Please add code

Comment (Luckygyana) [reply trainingmodel.py]:
```
(file1, file2) -> 0.35
(file3, file4) -> 0.4
(file5, file6) -> 0.7
(file8, file7) -> 0.8
```
We have file like this right?

File: trainingmodel.py (hunk 1 above)
Comment (Luckygyana) [reply trainingmodel.py]: I will change this

**Review by turian — APPROVED** [review]

**Pull request merged**